src/iperf_udp.c

```diff
@@ -183,7 +186,7 @@ iperf_udp_recv(struct iperf_stream *sp)
 	sp->jitter += (d - sp->jitter) / 16.0;
     }
     else {
-	if (sp->test->debug)
+	if (test->debug_level >= DEBUG_LEVEL_INFO)
 	    printf("Late receive, state = %d\n", sp->test->state);
```

Comment:
Maybe `sp->test->state` should be `test->state` for consistency with the other changes?

Reply:
Changed

Reply:
Thanks!